extensions/wikia/RTE/js/plugins/media/plugin.js

```diff
@@ -275,11 +275,18 @@ CKEDITOR.plugins.add('rte-media', {
 		});
 
 		// images / videos specific setup
-		image = media.filter('img.image');
-		self.setupImage(image);
+		// XW-4380: Filter out media inside templates, extension tags etc. which are not editable
+		media.get().filter(function (mediaNode) {
+			return mediaNode.isContentEditable;
+		}).forEach(function (mediaNode) {
+			if (mediaNode.classList.contains('image')) {
+				self.setupImage($(mediaNode));
+			}
 
-		video = media.filter('img.video');
-		self.setupVideo(video);
+			if (mediaNode.classList.contains('video')) {
```

Comment:
can one mediaNode be an image and video?
if not I'd do else if

Reply:
@TK-999 any idea if this is possible?